### PR TITLE
Create dedicated command to collect public cloud stats

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: ocw/management/commands/dumpstate.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
+from ocw.lib.dump_state import dump_state
+
+
+class Command(BaseCommand):
```
Comment (ricardobranco777) [inline ocw/management/commands/dumpstate.py:5]: I would use a less generic name for this class.

File: ocw/lib/influx.py [hunk 2]
```diff
@@ -53,5 +43,10 @@ def write(self, measurement: str, field: str, value: int) -> None:
             point = Point(measurement).field(field, value)
             try:
                 self.__client.write(bucket=self.bucket, org=self.org, record=point)
-            except (InfluxDBError, HTTPError, TimeoutError) as exception:
-                logger.warning(f"Failed to write to influxdb(record={point}): {exception}")
+            except (InfluxDBError, HTTPError) as exception:
+                logger.warning("Failed to write to influxdb(record=%s): %s", point, exception)
+
+    def dump_resource(self, provider: str, field: str, dump_method: Callable):
```
Comment (ricardobranco777) [inline ocw/lib/influx.py:49]:
```suggestion
    def dump_resource(self, provider: str, field: str, dump_method: Callable) -> None:
```

**Review by ricardobranco777 — DISMISSED** [review]
Couple of suggestions. LGTM.

File: ocw/management/commands/dumpstate.py (hunk 1 above)
Comment (asmorodskyi) [reply ocw/management/commands/dumpstate.py:5]: please check ALL other files in this folder :) I am not 100% sure but this is might be requirement from django how this "commands flow" is working . In any case no one except django itself suppose to create instances of this class so this one and only place where this name will be used

File: ocw/lib/influx.py (hunk 2 above)
Comment (asmorodskyi) [reply ocw/lib/influx.py:49]: make sense

**Pull request merged**